internal/config/config.go

```diff
@@ -10,37 +10,61 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package token
+package config
 
 import (
 	"errors"
+	"fmt"
 	"os"
 
-	"gopkg.in/yaml.v3"
+	"sigs.k8s.io/yaml"
 )
 
-type Tokens struct {
-	GithubToken string `yaml:"githubToken"`
-	JiraToken   string `yaml:"jiraToken"`
+const schemaName string = "gh2jira.config"
+const defaultJiraBaseURL string = "https://issues.redhat.com/"
+
+type tokenStore struct {
+	JiraToken   string `json:"jira"`
+	GithubToken string `json:"github"`
+}
+
+type Config struct {
+	Schema      string     `json:"schema"`
+	JiraBaseUrl string     `json:"jiraBaseUrl,omitempty"`
+	Tokens      tokenStore `json:"authTokens"`
+}
+
+func (c *Config) setDefaults() error {
+	if c.JiraBaseUrl == "" {
+		c.JiraBaseUrl = defaultJiraBaseURL
+	}
+	return nil
 }
 
-func ReadTokensYaml(file string) (*Tokens, error) {
-	data, err := readFile(file)
+func ReadFile(f string) (*Config, error) {
+	b, err := readFile(f)
 	if err != nil {
 		return nil, err
 	}
 
-	var tokens Tokens
-	err = yaml.Unmarshal([]byte(data), &tokens)
+	var c Config
```

Comment:
Alternatively, could write this as 
```suggestion
	var c Config = Config{
		JiraBaseUrl: defaultJiraBaseURL,
	}
```
and if the unmarshal has some other value it will override it. 

The current approach is fine.